documentation/automation.md

```diff
@@ -0,0 +1,50 @@
+# Volunteers for Salesforce Automation Inventory
+
+## Key Workflows
+
+This section describes the key workflows for developers, QEs, and technical
+writers.
+
+| Workflow                   | Flow                 | Org Type       | Managed | Namespace       | NPSP |
+| -------------------------- | -------------------- | -------------- | ------- | --------------- | ---- |
+| Development                | `dev_org`            | dev            |         |                 |      |
+| Development (Namespaced)   | `dev_org_namespaced` | dev_namespaced |         | `GW_Volunteers` |      |
+| QA                         | `qa_org`             | dev            |         |                 |      |
+| Regression                 | `regression_org`     | regression     | ✔       |                 |      |
```

Comment:
@davidmreed - Has this changed? Last time I tested using regression_org flow it had the GW_Volunteers__ namespace.